examples/react_example/src/Css/Genre.css

```diff
@@ -0,0 +1,35 @@
+.genre-btn {
+  background-color: transparent;
+  cursor: default;
```

Comment:
Could be a good idea to change from default to pointer. Makes it more intuitive and lets the users know that all of the genres are clickable.